shopify/resources/inventory_level.py

```diff
@@ -38,3 +43,10 @@ def set(cls, location_id, inventory_item_id, available, disconnect_if_necessary=
         }
         resource = cls.post('set', body=json.dumps(body))
         return InventoryLevel(InventoryLevel.format.decode(resource.body))
+
+    def is_new(self):
+        return False
+
+    def destroy(self):
+        options = {"inventory_item_id": self.inventory_item_id, "location_id": self.location_id}
+        return self.__class__.connection.delete(self._element_path(query_options=options), self.__class__.headers)
```

Comment:
just calling `self.connection...` and `self.headers...` just should just work, self is an instance of the `__class_` then it should have the same attributes.

Reply:
Looks good @mlhamel . Didn't have time to take a look at this before now, but had something similar going on.